.travis.yml

```diff
@@ -18,6 +18,14 @@ matrix:
       env: TOXENV=py36-local
     - python: 3.6
       env: TOXENV=py36-integ
+    - python: 3.7
+      env: TOXENV=py37-local
+      dist: xenial
+      sudo: true
+    - python: 3.7
+      env: TOXENV=py37-integ
+      dist: xenial
+      sudo: true
```

Comment:
What is this for?

Reply:
the short answer is that it's currently necessary to get 3.7 on Travis. https://github.com/travis-ci/travis-ci/issues/9815

Reply:
gotcha, thanks!